README.md

```diff
@@ -2,6 +2,160 @@
 
 Command-line interface for Jetfire/Transformations.
 
+## Usage
+
+### Authenticate
+
+To use `jetfire-cli`, the `JETFIRE_API_KEY` environment variable must be set to a valid API key for a service account which has access to Jetfire/Transformations.
+
+By default, `jetfire-cli` runs against the main CDF cluster (europe-west1-1).
+To use a different cluster, specify the `--cluster` parameter. Note that this is a global parameter, which must be specified before the subcommand.
+For example:
+
+```
+jetfire --cluster=greenfield <subcommand> [...args]
+```
+
+### Deploy transformations
+
+The primary purpose of `jetfire-cli` is to support continuous delivery, allowing you to manage transformations in a version control system.
+
+Transformations are described by YAML files, whose structure is described further below in this document.
+
+It is recommended to place these manifest files in their own directory, to avoid conflicts with other files.
+
+#### From the command line
+
+To deploy a set of transformations, use the `deploy` subcommand:
+
+```
+jetfire deploy <path>
+```
+
+The `<path>` argument should point to a directory containing YAML manifests.
+This directory is scanned recursively for `*.yml` and `*.yaml` files, so you can organize your transformations into separate subdirectories.
+
+#### From a GitHub Workflow
+
+`jetfire-cli` also provides a GitHub Action, which can be used to deploy transformations.
+To deploy a set of transformations in a GitHub workflow, add a step which references the action in your job:
+
+```yaml
+- name: Deploy transformations
+  uses: cognitedata/jetfire-cli@v1
+  with:
+    path: transformations
+    api-key: ${{ secrets.JETFIRE_API_KEY }}
+    # If not using the main europe-west1-1 cluster:
+    # cluster: greenfield
+  env:
+    SOME_API_KEY: ${{ secrets.SOME_API_KEY }}
+```
+
+This GitHub action takes the following inputs:
+
+| Name      | Description |
+|-----------|-------------|
+| `path`    | _(Required)_ The path to a directory containing transformation manifests. |
+| `api-key` | _(Required)_ The API key used for authenticating with transformations. Equivalent to setting the `JETFIRE_API_KEY` environment variable. |
+| `cluster` | _(Optional)_ The name of the cluster where Jetfire/Transformations is hosted. |
+
+Additionally, you must specify environment variables for any API keys referenced in transformation manifests.
+
+
+See [workflow.example.yml](workflow.example.yml) for a complete example.
+
+#### Manifest file format
+
+```yaml
+# Required
+externalId: my-transformation
+
+
+# Required
+name: My Transformation
+
+
+# Optional, default: false
+shared: true
+
+
+# Optional, default: null
+# If null, transformation will not be scheduled.
+schedule: "1 * * * *"
+
+
+# Required
+# The path to a file containing the SQL query for this transformation.
+query: my-transformation.sql
+
+
+# Required
+# Valid values are: assets, assethierarchy, events, timeseries, datapoints, stringdatapoints
```

Comment:
Also "sequences", "sequencerows", "files" now?

Reply:
For sure! I'll add that in a separate PR.